python_modules/libraries/dagster-docker/dagster_docker_tests/conftest.py

```diff
@@ -1,18 +1,54 @@
+import os
 from contextlib import contextmanager
+from pathlib import Path
 
+import boto3
 import pytest
-from dagster_aws.utils import ensure_dagster_aws_tests_import
+from dagster_test.fixtures import docker_compose_cm, network_name_from_yml
 
-ensure_dagster_aws_tests_import()
-from dagster_aws_tests.aws_credential_test_utils import get_aws_creds
+IS_BUILDKITE = os.getenv("BUILDKITE") is not None
+compose_file = Path(__file__).parent / "docker-compose.yml"
 
 
-@pytest.fixture
-def aws_env():
-    aws_creds = get_aws_creds()
-    return [
-        f"AWS_ACCESS_KEY_ID={aws_creds['aws_access_key_id']}",
-        f"AWS_SECRET_ACCESS_KEY={aws_creds['aws_secret_access_key']}",
+@pytest.fixture(scope="session")
+def postgres_network():
+    yield network_name_from_yml(compose_file)
+
+
+@pytest.fixture(scope="session")
+def hostnames():
+    with docker_compose_cm(compose_file) as hostnames:
+        yield hostnames
+
+
+@pytest.fixture(scope="session")
+def postgres_hostname(hostnames):
+    yield hostnames["postgres"]
+
+
+@pytest.fixture(scope="session")
+def aws_env(hostnames):
```

Comment:
I ended up not sharing these fixtures with #29205 since there a) wasn't a great place to put fixtures that cross both dagster-postgres and dagster-aws and b) we only do this in a few different places.